apps/storybook-react-native/stories/WalletHome.stories.tsx

```diff
@@ -0,0 +1,413 @@
+  Icon,
+  IconName,
+  IconColor,
+  Text,
+  TextButton,
+  TextColor,
+  TextVariant,
+  BadgeWrapperPosition,
+  BadgeStatus,
+  BadgeStatusStatus,
+  IconSize,
+} from '@metamask/design-system-react-native';
+import { useTailwind } from '@metamask/design-system-twrnc-preset';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { View, ScrollView, Pressable } from 'react-native';
+
+const meta: Meta = {
+  title: 'Examples/Wallet Home',
+  component: () => null,
+};
+
+export default meta;
+type Story = StoryObj;
+
+const WalletHome: React.FC = () => {
+  const tw = useTailwind();
+
+  return (
+    <ScrollView style={tw`bg-background-default flex-1`}>
+      {/* Container */}
+      <View style={tw`bg-background-default w-full py-4`}>
+        {/* Header */}
+        <View style={tw`border-border-muted border-b px-4 py-4`}>
+          <View style={tw`flex-row items-center justify-between`}>
+            <View style={tw`flex-1 flex-row items-center overflow-hidden`}>
+              <AvatarAccount
+                shape={AvatarBaseShape.Square}
+                variant={AvatarAccountVariant.Maskicon}
+                address=""
+              />
+              <View style={tw`ml-2 flex-1`}>
+                <Text variant={TextVariant.HeadingMd} numberOfLines={1}>
+                  DeFi Account
+                </Text>
+              </View>
+              <ButtonIcon
+                iconName={IconName.ArrowDown}
+                accessibilityLabel="Switch Account"
+              />
+            </View>
+            <View style={tw`ml-2 flex-row items-center`}>
+              <ButtonIcon iconName={IconName.Copy} accessibilityLabel="Copy" />
+              <ButtonIcon
+                iconName={IconName.Global}
+                accessibilityLabel="Network"
+              />
+              <BadgeWrapper
+                badge={<BadgeCount count={10} max={9} />}
+                position={BadgeWrapperPosition.TopRight}
+              >
+                <ButtonIcon
+                  iconName={IconName.Menu}
+                  accessibilityLabel="Menu"
+                />
+              </BadgeWrapper>
+            </View>
+          </View>
+        </View>
+
+        {/* Balance */}
+        <View style={tw`px-4 py-4`}>
+          <Text variant={TextVariant.DisplayMd}>$10,528.46</Text>
+          <View style={tw`flex-row items-center`}>
+            <Icon name={IconName.Arrow2Up} color={IconColor.SuccessDefault} />
+            <Text color={TextColor.TextAlternative}>$367.00 (+0.66%)</Text>
+          </View>
+        </View>
+
+        {/* Actions */}
+        <View style={tw`flex-row gap-3 px-4 py-4`}>
+          <ButtonBase twClassName="bg-background-muted h-20 flex-1 rounded-lg px-0 py-4">
+            <View style={tw`flex-col items-center justify-center`}>
+              <Icon name={IconName.Bank} />
+              <Text fontWeight={FontWeight.Medium}>Buy/Sell</Text>
+            </View>
```

Comment:
If we had access to the content wrapping View via a prop object we could use that instead but this works